examples/stage-2-shield-study/addon/bootstrap.js

```diff
@@ -126,6 +131,7 @@ async function chooseVariation() {
   return toSet;
 }
 
+// TODO marcrowo: Move to lib?
```

Comment:
I'd suggest ditching this helper altogether. I don't think it makes the API much easier, and adds extra complexity that the average add-on developer won't already know.